apps/main/src/llamalend/queries/market-rates.query.ts

```diff
@@ -0,0 +1,23 @@
+import { getLlamaMarket } from '@/llamalend/llama.utils'
+import { LendMarketTemplate } from '@curvefi/llamalend-api/lib/lendMarkets'
+import { type MarketQuery, queryFactory, rootKeys, MarketParams } from '@ui-kit/lib/model'
+import { marketIdValidationSuite } from '@ui-kit/lib/model/query/market-id-validation'
+import type { Decimal } from '@ui-kit/utils'
+
+export const { useQuery: useMarketRates } = queryFactory({
```

Comment:
this query already exists
https://github.com/curvefi/curve-frontend/blob/29e90a91cb488f909bea08b9028935ef10d5214f/apps/main/src/llamalend/features/borrow/queries/borrow-apy.query.ts#L35